src/components/PDFViewer/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// TODO: get from site branding
+import settings from '../../data/configuration/constants';
+
+import './PDFViewer.scss';
+
+const getPDFUrl = relativeUrl => (
+  `${settings.journalsBackendBaseUrl}${relativeUrl}`
+);
+
+const getIFrameUrl = relativeUrl => (
+  `${settings.journalsBackendBaseUrl}/static/pdf_js/web/viewer.html?file=${getPDFUrl(relativeUrl)}`
+);
+
+const PDFViewer = props => (
+  // If the title is an empty string do not display one
+  <span id={props.id}>
+    <div className="pdf-border">
+      { (props.title.trim() !== '') ? <div className="image-title">{props.title} </div> : '' }
+      <div className="pdf-block embed-responsive embed-responsive-1by1">
```

Comment:
Nice find on these classes! I hadn't seen them before.

We can either do the following in this PR, or during prettification later:
I think we're going to want to want to change the ratio by viewport size. 1x1 might be too short for phones, and too tall for laptops. We may want to use a custom class and set the ratio based on screen size. E.g.

```
/* small screen */
@media (max-width: 576px){
  responsive-pdf::before{
    padding-top: 133.33%; /*4:3*/
  }
}
```

for each of the sizes. 

Again, doesn't have to be done in this PR if you don't want.

Reply:
thank you! i wasn't really sure how to implement custom classes like this